tools/scripts/artifacts/utils.ts

```diff
@@ -12,51 +12,64 @@ export class Utils {
   public static readonly EMPTY_GITHUB_COMMENTS =
     'No snapshots of projects have been published (probably no project is affected)';
 
+  public static globProjectJSON(): string[] {
+    const projects = execSync(
+      `npx glob --cwd=${process.cwd()} --absolute --dot ./**/project.json`
+    )
+      .toString()
+      .trim();
+    return projects.split('\n').filter((projectJSON) => {
+      return (
+        !projectJSON.includes('dist') &&
```

Comment:
Not 100% sure about this filter. This would ignore also projects with `dist` in their name. Would it be possible to instead look for dist folders a bit more explicitly, like looking for `/dist/`? 🤔